main.c

```diff
@@ -184,6 +184,7 @@ STATIC void stop_mp(void) {
     #endif
 
     background_callback_reset();
+    usb_background_schedule();
```

Comment:
I am not sure that we could just run `usb_background()` here, if it is then, we could remove the need to add this usb_background_schedule()

Reply:
executing `usb_background()` right away can remove the need for introducing `usb_background_schedule()`. However, I am not entirely sure at this point of reload (stopping vm), it is appropriate to run background task. There is bunch of init() afterwards. Therefore I choose the safer option to schedule it. I will leave the decision for you since you understand this better.